x-pack/test/functional/apps/lens/group4/share.ts

```diff
@@ -51,50 +62,20 @@ export default function ({ getService, getPageObjects }: FtrProviderContext) {
     });
 
     it('should enable both download and URL sharing for valid configuration', async () => {
-      await PageObjects.lens.clickShareMenu();
-
-      expect(await PageObjects.lens.isShareActionEnabled('csvDownload'));
-      expect(await PageObjects.lens.isShareActionEnabled('permalinks'));
-    });
-
-    it('should provide only snapshot url sharing if visualization is not saved yet', async () => {
-      await PageObjects.lens.openPermalinkShare();
-
-      const options = await PageObjects.lens.getAvailableUrlSharingOptions();
-      expect(options).eql(['snapshot']);
-    });
-
-    it('should basically work for snapshot', async () => {
-      const url = await PageObjects.lens.getUrl('snapshot');
-      await browser.openNewTab();
-
-      const [lensWindowHandler] = await browser.getAllWindowHandles();
-
-      await browser.navigateTo(url);
-      // check that it's the same configuration in the new URL when ready
-      await PageObjects.lens.waitForVisualization('xyVisChart');
-      expect(await PageObjects.lens.getDimensionTriggerText('lnsXY_yDimensionPanel')).to.eql(
-        'Average of bytes'
-      );
-      await browser.closeCurrentWindow();
-      await browser.switchToWindow(lensWindowHandler);
-    });
-
-    it('should provide also saved object url sharing if the visualization is shared', async () => {
-      await PageObjects.lens.save('ASavedVisualizationToShare');
-      await PageObjects.lens.openPermalinkShare();
+      await PageObjects.lens.clickShareModal();
 
-      const options = await PageObjects.lens.getAvailableUrlSharingOptions();
-      expect(options).eql(['snapshot', 'savedObject']);
+      expect(await PageObjects.lens.isShareActionEnabled('export'));
+      expect(await PageObjects.lens.isShareActionEnabled('link'));
     });
 
-    it('should preserve filter and query when sharing', async () => {
+    xit('should preserve filter and query when sharing', async () => {
```

Comment:
Why skipping this?

Reply:
Check out https://github.com/elastic/kibana/issues/181066 thanks!